### DISCUSSION TITLE
Docker Container

### DISCUSSION THREAD

**Discussion by kaaax0815** [envault/envault#53] [Ideas]
I have a Server with [CapRover](https://caprover.com/) which makes using Docker Containers easy (but doesn't support Docker Compose)
But to use it I need a Docker Image and I am not really familiar to Dockerfiles.

Could somebody make a Dockerfile or even a official Container on [Docker Hub](https://hub.docker.com/)

**Comment by danharrin** [comment]
Hey, check [this repo](https://github.com/carmino-app/envault-docker).
